02-bootstrap.md

```diff
@@ -299,9 +299,6 @@ This part's tricky. Drupal lazily starts sessions at the end of the request, so
 
 ```php
 session_id(drupal_random_key());
-$insecure_session_name = substr(session_name(), 1);
```

Comment:
I pulled these lines out because they are somewhat confusing and only pertain to HTTPS (they are wrapped in `if ($is_https && variable_get('https', FALSE)) {}` within the source code). It seems like the main idea is conveyed with the `session_id()` setting.